go.mod

```diff
@@ -17,9 +17,9 @@ require (
 	github.com/spf13/cobra v1.7.0
 	google.golang.org/grpc v1.57.0
 	google.golang.org/protobuf v1.30.0
-	k8s.io/api v0.27.0
-	k8s.io/apimachinery v0.27.0
-	k8s.io/client-go v0.27.0
+	k8s.io/api v0.28.0
+	k8s.io/apimachinery v0.28.0
+	k8s.io/client-go v0.28.0
```

Comment:
Any reason why component-base and component-helpers are not updated to 0.28.0?

Reply:
The script seems to have missed it earlier, but I did not get any error on updating them so should be good.